core/dbt/tests/util.py

```diff
@@ -85,3 +90,59 @@ def read_file(*paths):
     with open(os.path.join(*paths), "r") as fp:
         contents = fp.read()
     return contents
+
+
+def get_artifact(*paths):
+    contents = read_file(*paths)
+    dct = json.loads(contents)
+    return dct
+
+
+# For updating yaml config files
+def update_config_file(updates, *paths):
+    current_yaml = read_file(*paths)
+    config = yaml.safe_load(current_yaml)
+    config.update(updates)
+    new_yaml = yaml.safe_dump(config)
+    write_file(new_yaml, *paths)
+
+
+def run_sql(sql, adapter, database, schema, fetch=None):
```

Comment:
how is this related to run_sql at [tables.py](https://github.com/dbt-labs/dbt-core/blob/8fd8dfcf74657ccc5314640b22cb2589eed8a824/core/dbt/tests/tables.py#L311)

Reply:
This one has been renamed to run_sql_with_adapter now. It does pretty much the same thing as in tables.py, but the plan is to get rid of tables.py when we can, so I'd rather leave refactoring it to that effort.